src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -330,7 +301,7 @@ private void CreateIndicator()
                 Size = new Size2D((int)Style.IndicatorSize.Width, (int)Style.IndicatorSize.Height),
                 Opacity = 0.5f,
             };
-            indicator.Position = new Position2D((int)(Style.IndicatorSize.Width + Style.IndicatorSpacing) * indicatorList.Count, 0);
+            indicator.Position = new Vector2((int)(Style.IndicatorSize.Width + Style.IndicatorSpacing) * indicatorList.Count, 0);
```

Comment:
Note that, CA2000 warning is raised on implicit operator of `Position2D`. when you assign `Position2D` to `Position` Property, implicit operator is called. (https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Position2D.cs#L313)
In this implicit operator, the `position2D` object passed as parameter is not disposed.
CA2000 will be fixed with following code. however, since it is related to dali life-cycle, i'm not sure this is proper fix.
```diff
diff --git a/src/Tizen.NUI/src/public/Position2D.cs b/src/Tizen.NUI/src/public/Position2D.cs
index 48f2595f7..65f696754 100755
--- a/src/Tizen.NUI/src/public/Position2D.cs
+++ b/src/Tizen.NUI/src/public/Position2D.cs
@@ -312,7 +312,9 @@ namespace Tizen.NUI
         /// <since_tizen> 3 </since_tizen>
         public static implicit operator Vector2(Position2D position2d)
         {
-            return new Vector2((float)position2d.X, (float)position2d.Y);
+            var ret = new Vector2((float)position2d.X, (float)position2d.Y);
+            position2d.Dispose();
+            return ret;
         }

         /// <summary>
```

So, I think this kind of CA2000 should be handled in `Position2D`.

Reply:
The position2d is from method parameter, so I think the disposal should be handled by the user that passed the argument.

Reply:
```
Position2D p = new Position2D(10, 10);
Vector2 v1 = p;
Vector2 v2 = p;
Vector2 v3 = p;
```

Reply:
Thank you for reviewing it.
I wondered if I would change the operator in `Position2D`.
However, `Position` type of `Visual` is `Vector2`, so I changed the same here.

This patch goes like this,
and I will consider again where the similar warning occurs next time.